src/index.html

```diff
@@ -15,8 +15,89 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a href="index.html">
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a
+              href="index.html"
+              class="nav_link is-active"
+            >
+              Apple
+            </a>
+          </li>
```

Comment:
We need to add some 'breathing-space' here. 
![image](https://github.com/mate-academy/layout_moyo-header/assets/57567908/9dae0d44-70d7-457f-a8d2-8b2a77095568)